thoth-export-server/src/xml/onix21_ebsco_host.rs

```diff
@@ -399,14 +399,14 @@ impl XmlElementBlock<Onix21EbscoHost> for Work {
                         w.write(XmlEvent::Characters("Open access"))
                             .map_err(|e| e.into())
                     })?;
-                    // EBSCO Host require the price point for Open Access titles to be listed as "0.01 USD".
+                    // EBSCO Host require the price point for Open Access titles to be listed as "0.00 USD".
```

Comment:
add `and ProQuest Ebrary`

Reply:
I had misunderstood the requirements. I thought I had to update the EBSCO output, but it seems that the change is needed for ProQuest – so I'll leave the EBSCO output as it is and create a new one for ProQuest, which will be exactly the same except for the price point...